### PR TITLE
Add general properties to jdbc based connectors

### PR DESCRIPTION
**Pull request by agrawalreetika**
## Description
Add general properties to JDBC-based connectors

## Motivation and Context
Refactor and add available properties to existing Jdbc-based connector documents

## Impact
Provide more concrete details to users about available generic properties in Jdbc-based connector

## Contributor checklist

- [ ] Please make sure your submission complies with our [development](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#development), [formatting](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#formatting), [commit message](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#commit-formatting-and-pull-requests), and [attribution guidelines](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#attribution).
- [ ] PR description addresses the issue accurately and concisely.  If the change is non-trivial, a GitHub Issue is referenced.
- [ ] Documented new properties (with its default value), SQL syntax, functions, or other functionality.
- [ ] If release notes are required, they follow the [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines).
- [ ] Adequate tests were added if applicable.
- [ ] CI passed.

## Release Notes
```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-docs/src/main/sphinx/connector/mysql.rst [hunk 1]
```diff
@@ -48,6 +48,24 @@ with a different name (making sure it ends in ``.properties``). For
 example, if you name the property file ``sales.properties``, Presto
 will create a catalog named ``sales`` using the configured connector.
 
+General Configuration Properties
+---------------------------------
+
+================================================== =========================================================== ===========
+Property Name                                      Description                                                  Default
+================================================== =========================================================== ===========
+``user-credential-name``                           Name of the extra credentials property, whose value to use
```
Comment (steveburnett) [inline presto-docs/src/main/sphinx/connector/mysql.rst:57]: “Name of the extra credentials property, whose value to use as the user name.”
I’m not sure I understand what is meant here. Could you phrase this a different way, or maybe help me understand what is the meaning here and I can try to rephrase it? 

Note: I have the same question for this property in the other four files, and for password-credential-name in the five files, so I'd ask that whatever changes result from this comment be made on all ten lines. 
I suggest that we figure out what the change should be here. I could open ten identical comments but that feels messy and unhelpful.

Comment (agrawalreetika) [reply presto-docs/src/main/sphinx/connector/mysql.rst:57]: Thanks for the review @steveburnett. Here extra credentials property refers to the one mentioned here https://prestodb.io/docs/current/installation/jdbc.html#parameter-reference, It is to set username & password to catalogs via extra credentials for the catalog.

Example -
Jdbc catalog properties in Presto :
```
user-credential-name=jdbc_user
password-credential-name=jdbc_password
``` 
Then via presto jdbc connetion url we can set these username and password -
``` jdbc:presto://<presto>/<catalog>?extraCredentials=jdbc_user:<my_user>;jdbc_password:<my_password> ```

Will it make more clear to rephrase it something with the reference link - "Name of the extra credentials property, whose value to use as the user name, check the details of extra credentials property in https://prestodb.io/docs/current/installation/jdbc.html#parameter-reference"

If you have any better idea to rephrase it, please let me know

Comment (steveburnett) [reply presto-docs/src/main/sphinx/connector/mysql.rst:57]: Thanks very much @agrawalreetika! Your explanation and the link helped me understand what this pair is, and what it does. I think the link will help users understand, so yes I'd like to include it in this table. 

My suggestion is to replace 

Name of the extra credentials property, whose value to use as the user name.

with

Name of the `extraCredentials` property that the value is the JDBC driver user name. 
See `extraCredentials` in `Parameter Reference </installation/jdbc.html#parameter-reference>`_.

and replace 

Name of the extra credentials property, whose value to use as the password.

with 

Name of the `extraCredentials` property that the value is the JDBC driver user password. 
See `extraCredentials` in `Parameter Reference </installation/jdbc.html#parameter-reference>`_.

If my rephrasing does not make sense to you, please change it as you see appropriate. 

I tested the formatting of the link in my local presto-docs build and it works for me, I hope the copy-paste of the link works for you.

**Review by steveburnett — APPROVED** [review]
LGTM! (docs)

**Review by tdcmeehan — APPROVED** [review]

**Pull request merged**